src/pipeline_manager.py

```diff
@@ -156,7 +191,7 @@ def visualize(pipeline_name, image_dir=None, single_image=None, n_files=16, show
 
     images_with_drawn_boxes = pipeline.fit_transform(data)
```

Comment:
@taraspiotr @i008 this should be .transform right?

Reply:
@jakubczakon  ah, yes, we found that last night, I will change it in this PR